sklearn/naive_bayes.py

```diff
@@ -326,6 +326,8 @@ def _update_mean_variance(n_past, mu, var, X, sample_weight=None):
         # Compute (potentially weighted) mean and variance of new datapoints
         if sample_weight is not None:
             n_new = float(sample_weight.sum())
+            if n_new == 0:
+                return 0, 0
```

Comment:
I think this should be `mu` and `var` given the docstring of the method. (If `n_new == 0`, then the original mu and var should be returned?)

If we can not come up with a fix, then we can xfail this with a estimator tag. This way we can get the common test in first.